src/middleware.ts

```diff
@@ -106,3 +107,84 @@ export const withWebhookSignatureVerified =
 
     return handler(request);
   };
+
+export interface DashboardTokenPayload extends JwtPayload {
+  app: string;
+}
+
+export const withJWTVerified =
+  (getAppId: (request: Request) => Promise<string | undefined>): Middleware =>
```

Comment:
As I understand, that's because you decided to not include GraphQL query for even that simple purpose in SDK, just abstracting that away and requiring template to inject that logic here.

The same way we did that with @witoszekdev in #8 ?

Reply:
yup, we didn't want to include codegen in that repository in order to avoid the need of releases whenever Saleor Core API changes.

Reply:
Additional bonus - the app's author can implement caching of the app id in provided function. Avoiding additional call to the GQL API would improve performance.